ui/.gitignore

```diff
@@ -1,5 +1,23 @@
-.idea
-.DS_Store
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
 /node_modules
-/dist
-*.log
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
```

Comment:
- [ ] add `*.iml`

Reply:
It's already in `.gitignore`, but I can add it to `ui/.gitignore` if you want.

Reply:
no need to add `ui/.gitignore` since we have `.gitignore` :)